packages/contracts-bedrock/src/L1/OptimismPortal2.sol

```diff
@@ -125,6 +125,9 @@ contract OptimismPortal2 is Initializable, ResourceMetering, ReinitializableBase
     /// @notice Whether the OptimismPortal is using Super Roots or Output Roots.
```

Comment:
We may need to break this out into its own contract pending some open decisions on the rollout, will follow up as soon as possible